script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -547,58 +614,74 @@ Even though you possess such rare Mystic Eyes, there's a limit to how much of a
+ Call me a peace-loving idiot, but isn't it amazing that we've maintained this degree of public order for so long?
 }
 [sha:8719596211f8ffffa7e2cc93558d046c5e037b11]{
 -- Page 473, Offset 17747.
 -- 「……分かった。あんまり言いたくなかったけど、ちゃんと説明してあげる」
-"...Fine. I didn't want to do this, but I'll explain this properly to you."
+"...Fine. I didn't want to tell you all this, but let me explain things to you properly."
 }
 [sha:d0a2294f9f02157f501b5cd984f71352cfc59197]{
 -- Page 474, Offset 17748.
 -- 「いい？　志貴は生物の死は視やすくて、鉱物の死は視にくいって言ってたでしょ？
-"Listen. I told you that it's easy to see the death of living things and that it's difficult to see the death of minerals, right?
+"Are you ready?
+ Shiki, you said that you could make out the lines of death on living things
+ a lot easier than you could on inanimate objects, right?
 }
 [sha:df58b677bafbe58419434a3ea9409ccc15a8ddf1]{
 -- Page 474, Offset 17749.
 -- それはね、結局はあなたの脳の回線の問題なの。
-That's because it's a problem of the circuits in your brain.
+The reason for that all comes down to the circuits in your brain.
 }
 [sha:c700a8b0b5c4ce6c2a7f4804d814102d178cd544]{
 -- Page 474, Offset 17750.
 -- モノの死。その因果を視る、という事は、実際は視ているではなく読んでいる、という意味合いに近いわ」
-The death of things. When you observe its causality, what you're actually doing is not seeing, but something closer to that of reading."
+The death of things.
+ We say that you're 'seeing' the fate of an object, but in reality it's a bit more nuanced;
+ It's more like you're 'reading' it than 'seeing' it."
 }
 [sha:3cad7ac871256e275b1c8e8982a196bda200db39]{
 -- Page 475, Offset 17751.
 -- 「すべての事象には大本になった原因、絶対の一がある。
-// Type moon lore experts, take the wheel, i am DONE. i am also DONE.
-"All events have a cause as their foundation-–an absolute first.
+// Type moon lore experts, take the wheel, i am DONE. i am also DONE. I am only just beginning >:)
+"All events have a cause that becomes their foundation, an absolute first.
 }
 [sha:91a378903530dce68e2051c0deada9ff850506c7]{
 -- Page 475, Offset 17752.
 -- この宇宙のはじまりにしておしまいの地点。
-The beginning of the universe, as well as its final end point.
+The point where this universe begins, and the point where it ends.
 }
 [sha:dae90a2dace9f7f388e7840b68e690cb0d7e1a34]{
 -- Page 475, Offset 17753.
 -- そこには“全てを記録したモノ”がある。記録というよりは“有る”ものだから、情報とは呼べない。ただ“有る”だけ。それ自体に意思はないし、方向性もありえない。原因を垂れ流しているだけの、根源の渦みたいなものね」
-Within it is an 'archive of all things.' But rather than an archive, it is closer to 'existence' itself, and therefore cannot be called information. It is simply 'existence.' That in itself holds no purpose, nor does it have directivity. All it does is release causes, like the Swirl of the Root.."
+Between two, there exists an 'archive of all things'.
+ Though it's called an archive, what it contains can't be called 'information' so much as 'existence'.
+ 'Existence' and nothing more.
+ It has no will of its own, no overarching desires.
+ All it does is release causes, like the Swirl of the Root."
```

Comment:
Nasu read the encyclopedia entry for the bhagavad gita when he was 16 and has been trying his god damn hardest ever since